cpp/src/Slice/Grammar.y

```diff
@@ -1537,32 +1564,39 @@ interface_list
     StringTokPtr scoped = StringTokPtr::dynamicCast($1);
     ContainerPtr cont = unit->currentContainer();
     TypeList types = cont->lookupType(scoped->v);
-    if(!types.empty())
+    if (!types.empty())
     {
         InterfaceDeclPtr interface = InterfaceDeclPtr::dynamicCast(types.front());
-        if(!interface)
+        if (!interface)
         {
-            string msg = "`";
-            msg += scoped->v;
-            msg += "' is not an interface";
-            unit->error(msg); // $$ is a dummy
+            if (auto alias = TypeAliasPtr::dynamicCast(types.front()))
```

Comment:
Same here, we can get rid of the nested if

`if (!interface && auto alias = ...`

Reply:
Yep, fixed same as above.